requirements.txt

```diff
@@ -1,20 +1,17 @@
-configobj==5.0.5
-cherrypy==3.6.0
-#python-ldap==2.3.13
-ws4py==0.3.2
-SQLAlchemy==1.0.0
-six==1.5.2
-Jinja2==2.6
-rpctools==0.2.1
-logging_unterpolation==0.2.0
-requests==2.2.1
-pytest==3.0.1
-mock==1.0.1
-Sphinx==1.2.1
-coverage==3.6
-paver==1.2.2
-wheel==0.24.0
-pip==1.5.6
-sh==1.09
-python-prctl==1.6.1
-psutil==4.3.0
\ No newline at end of file
+configobj>=5.0.5
+cherrypy>=3.6.0
+ws4py>=0.3.2
+SQLAlchemy>=1.0.0
+six>=1.5.2
+Jinja2>=2.6
+rpctools>=0.3.1
+logging_unterpolation>=0.2.0
+requests>=2.2.1
+pytest>=3.0.1
+mock>=1.0.1,<1.1
+Sphinx>=1.2.1
+coverage>=3.6
+paver>=1.2.2
+wheel>=0.24.0
+pip>=1.5.6
+sh>=1.09
```

Comment:
See https://github.com/magfest/sideboard/issues/25 for a discussion of this change.  The tldr is that pinning dependencies is good for applications and bad for libraries.  We don't want to cause an error when a plugin pins a different version of a dependency than Sideboard itself does.